lib/ex_plasma/output/type/payment_v1.ex

```diff
@@ -97,8 +97,9 @@ defmodule ExPlasma.Output.Type.PaymentV1 do
   defp do_validate([_output_guard, nil, _amount]), do: {:token, :cannot_be_nil}
   defp do_validate([nil, _token, _amount]), do: {:output_guard, :cannot_be_nil}
   defp do_validate([@zero_address, _token, _amount]), do: {:output_guard, :cannot_be_zero}
+  defp do_validate([<<_::160>>, _token, _amount]), do: :ok
 
-  defp do_validate([_, _, _]), do: :ok
+  defp do_validate([_, _, _]), do: {:output_guard, :invalid_length}
```

Comment:
what happened here? it used to be OK now it's not :))

Reply:
yep, the current output validation is very permissive, I don't want to change it for now until I do #69, but this one is just a small exception :p I added a check for the output guard length, if the length is wrong, then it's not OK